### ISSUE TITLE
1.18.2-4.0.0 - Invalid mixin target forge [CONFIRMED]  

### ISSUE THREAD

**Issue by border999** [jtorleonstudios/HUB-Library-Ferret#12]
[debug.log](https://github.com/jtorleonstudios/HUB-Library-Ferret/files/10962875/debug.log)
[latest.log](https://github.com/jtorleonstudios/HUB-Library-Ferret/files/10962876/latest.log)

**Comment by border999** [comment]
quick look says something about mixin injection failure and world generation

**Comment by jtorleonstudios** [comment]
Ok I will try and fix it in priority, please provide me the list of mods

**Comment by jtorleonstudios** [comment]
I haven't tried yet but I think I understand where the problem comes from, the library was compiled for version 40.1.73, and your version is 4.2.0, I'll confirm more once the test is done

**Comment by border999** [comment]
> Ok I will try and fix it in priority.
> 
> Please provide me the list of mods in your pack

that list is TOO long. and it looks like you've already found the issue. If a second issue arises after this one is fixed I will open a new issue.

**Comment by Warhand** [comment]
> I haven't tried yet but I think I understand where the problem comes from, the library was compiled for version forge-40.1.73, and your version is forge-4.2.0 (So my theory for the moment is that the code has changed between the 2 versions mentioned, which makes the mixin invalid)
> 
> I'll confirm more once the test is done

I tested this on a profile with no mods other than library ferret on forge 40.2.0 and also forge 40.1.73

**Comment by jtorleonstudios** [comment]
> > I haven't tried yet but I think I understand where the problem comes from, the library was compiled for version forge-40.1.73, and your version is forge-4.2.0 (So my theory for the moment is that the code has changed between the 2 versions mentioned, which makes the mixin invalid)
> > I'll confirm more once the test is done
> 
> I tested this on a profile with no mods other than library ferret on forge 40.2.0 and also forge 40.1.73, and encountered the same crash on both.

**Thanks for your feedback,** a patch should be out in the day. 

_If you haven't already done so, use the old version while waiting for the patch, make sure you retrograde all the necessary mods_

**Comment by jtorleonstudios** [comment]
A version will be available on Curseforge in a few minutes, it fixes the issue.

The problem was that I had forgotten to add the reference "refmap.json" in my mixins.json... 

Everything should work now, I close the issue, do not hesitate to open a new for any other problem

Thanks you for your report, and good game
